src/components/timeline/TrackNetwork.js

```diff
@@ -185,18 +191,25 @@ class NetworkCanvas extends PureComponent<CanvasProps> {
         const path = new Path2D();
         path.moveTo(start, y);
         path.lineTo(end, y);
-        if (timing.index[timingIndex] === hoveredMarkerIndex) {
-          // Save the hovered path to draw it at the end, on top of everything else.
```

Comment:
optional nit: I think it would be valuable to keep this comment still. What do you think?

Reply:
I rewrote it :-)